.github/workflows/frontend-tests.yml

```
@@ -0,0 +1,50 @@
+      uses: actions/upload-artifact@v3
+      with:
+        name: coverage-report
+        path: coverage/
+      
+    - name: Check test coverage
+      run: |
+        COVERAGE=$(npm test -- --coverage --coverageReporters="json-summary" | grep "All files" | awk '{print $4}' | tr -d '%')
+        if (( $(echo "$COVERAGE < 70" | bc -l) )); then
+          echo "Test coverage is below 70%"
+          exit 1
+        fi 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)